tools/pretty-json.py

```diff
@@ -15,11 +15,11 @@
 # read as JSON and normalize to standard formatting
 with open(arguments.filename, 'r') as f:
     j = json.loads(f.read())
-s = json.dumps(j, indent = 2)
+s = json.dumps(j, indent=2)
 
 # fold arrays of scalar values into one line
-s = re.sub(r'\[\n +(\"[A-Za-z-]+\"|[\d\.-]+)', r'[\1', s)
-s = re.sub(r',\n +(\"[A-Za-z-]+\"|[\d\.-]+)(?=[,\n])', r', \1', s)
+s = re.sub(r'\[\n +(\"[A-Za-z-]+\"|[\d-]+)', r'[\1', s)
```

Comment:
Isn't this changing the semantics? `[\d\.-]` was supposed to match digits, dashes and dots, `[\d-]` now misses the dots, no?

Reply:
> Isn't this changing the semantics? `[\d\.-]` was supposed to match digits, dashes and dots, `[\d-]` now misses the dots, no?

Yes you are right. I removed to much. I should have just removed the escape character, as it isn't needed inside of the character class []. Thanks!